packages/polar/src/lib/deploy/contract.ts

```diff
@@ -259,7 +259,7 @@ export class Contract {
   async queryMsg (
     methodName: string,
     callArgs: object // eslint-disable-line @typescript-eslint/ban-types
-  ): Promise<any> { // eslint-disable-line  @typescript-eslint/no-explicit-any
+  ): Promise<string> {
```

Comment:
could you please confirm if this returns string ?

Reply:
is it string ?

Reply:
Couldn't confirm. Have changed type to any.